constitution.md

```diff
@@ -87,6 +87,8 @@ The Executive Board will meet as a body at least once per month. Every meeting m
 
 The Co-Chair of Operations is responsible for scheduling, organizing, and running all Executive Board meetings, including taking attendance of present members and appointing an officer to take minutes.
 
+// Can members attend those meetings ??
```

Comment:
Love that we can comment on specific lines. 

I can't think of any either. Perhaps we should be explicit about that?